examples/clusterscoped/escape-to-host-si-sib-wthout-exclusion.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: intent.security.nimbus.com/v1
+kind: SecurityIntent
+metadata:
+  name: escape-to-host
+spec:
+  intent:
+    id: escapeToHost
+    description: "A attacker can breach container boundaries and can gain access to the host machine"
+    action: Block
+---
+apiVersion: intent.security.nimbus.com/v1
+kind: ClusterSecurityIntentBinding
+metadata:
+  name: escape-to-host-binding
+spec:
+  intents:
+    - name: escape-to-host
+  selector:
+    resources:
+
+      - kind: Pod
+        namespace: ns3
+        matchLabels:
+          app: db
+          database: mongo
+            
+
+      - kind: Deployment
```

Comment:
In case we are using PSA, then there is autogen of rules pod controllers. We really do not need to have deployment, and stateful sets.